src/test/scala/scalismo/mesh/MeshMetricsTests.scala

```diff
@@ -51,7 +51,7 @@ class MeshMetricsTests extends ScalismoTestSuite {
 
     it("should be (slightly) lower than the average translation applied to each vertex") {
       MeshMetrics.avgDistance(mesh, translatedMesh) should be < translationLength.toDouble
-      MeshMetrics.avgDistance(mesh, translatedMesh) should be(translationLength.toDouble +- (translationLength * 0.2))
+      MeshMetrics.avgDistance(mesh, translatedMesh) should be(translationLength.toDouble +- (translationLength * 0.3))
```

Comment:
I don't understand why the margin needs to be increased here. Shouldn't the distance between the meshes now become smaller and not larger, because the distance to the closest point on the vertex is always at least as long as the distance to the closest point on the surface?

Reply:
@Andreas-Forster @madsendennis What is the status of this PR? Can it be merged and used in release-0.17 or shall we wait until the next release for a more complete solution?

Reply:
I think this PR makes sense and improves the current state. So yes I would argue for merging this PR. The "abuse" of the discussion on the issue #252 is more meant as an effort to continue to work towards a metric for users that are not aware of the influence of the discretization of the meshes.